.github/workflows/ci.yml

```
@@ -66,7 +66,7 @@ jobs:
       - name: Run cargo checks
         run: |
           set -x
-          VERSIONS=("3.7" "3.8" "3.9" "3.10")
+          VERSIONS=("3.7" "3.8" "3.9" "3.10" "3.10" "3.11")
```

Looks like 3.10 unintentionally duplicated

```suggestion
          VERSIONS=("3.7" "3.8" "3.9" "3.10" "3.11")
```